Signing.props

```diff
@@ -11,27 +11,20 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)\Keys\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
-  <!-- Default metadata for all files to sign -->
-  <ItemDefinitionGroup>
-    <FilesToSign>
-      <!-- Microsoft authenticode certificate for all file types -->
-      <Authenticode Condition="'%(FilesToSign.Authenticode)' == ''">Microsoft</Authenticode>
-      <!-- Microsoft strong-name key for managed assemblies -->
-      <StrongName Condition="'$(TargetExt)' == '.winmdobj' OR '$(TargetExt)' == '.dll' OR '$(TargetExt)' == '.exe'">MsSharedLib72</StrongName>
-    </FilesToSign>
-  </ItemDefinitionGroup>
+  <Target Name="AddToSign" AfterTargets="CopyFilesToOutputDirectory" >
 
-  <ItemGroup>
-    <!-- Automatically sign binary generated by the project that imports Signing.props -->
-    <FilesToSign Include="$(TargetPath)" />
-  </ItemGroup>
-
-  <!-- Sign PowerShell scripts and modules copied to the output directory of the project -->
-  <Target Name="PreparePowerShellScriptsForSigning" BeforeTargets="SignFiles">
     <ItemGroup>
-      <FilesToSign Include="$(OutputPath)\**\*.ps*1">
+      <!-- Automatically sign binary generated by the project that imports Signing.props and all ps1 files -->
+      <FilesToSign Include="$(TargetPath)" >
+        <Authenticode>Microsoft</Authenticode>
+        <StrongName>MsSharedLib72</StrongName>
+      </FilesToSign>
+      <FilesToSign Include="$(OutputPath)*.ps1">
```

Comment:
Just to be safe, does $(OutputPath) end in a '\' and all the powershell scripts are in that folder such that the ** wildcard is not needed?